.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+{
```

Comment:
JSON doesn't allow comments 🤔 should this have a `jsonc` file-extension ? (is that the official extension for it?)

Reply:
Unfortunately, the official [devcontainers spec](https://containers.dev/implementors/spec/) defines the file as a JSON with Comments, but using the  `.json` extension. I know.

Reply:
😬 that feels very "YOLO - works for us, just not for everyone else"